pkg/kv/kvserver/metrics.go

```diff
@@ -809,6 +809,18 @@ bytes preserved during flushes and compactions over the lifetime of the process.
 		Measurement: "Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
+	metaBlockLoadsInProgress = metric.Metadata{
```

Comment:
Do we want another metric for time waited? I guess we also have a _total_ block load time. No strong opinion.

Reply:
Not entirely sure what would be useful. The sum of wait times for each load? Or the total amount of time during which there was a non-empty queue? Probably the latter. Either way, that would have to be added to the semaphore so I won't do it in as part of this patch.